control/autoware_simple_pure_pursuit/src/simple_pure_pursuit.cpp

```diff
@@ -63,6 +63,18 @@ void SimplePurePursuitNode::on_timer()
 autoware_control_msgs::msg::Control SimplePurePursuitNode::create_control_command(
   const Odometry & odom, const Trajectory & traj)
 {
+  // Check for empty trajectory
+  if (traj.points.empty()) {
+    RCLCPP_WARN_THROTTLE(
+      get_logger(), *get_clock(), 5000,
+      "Received empty trajectory, returning zero velocity command.");
+    autoware_control_msgs::msg::Control cmd;
+    cmd.stamp = odom.header.stamp;
+    cmd.longitudinal.velocity = 0.0;
+    cmd.longitudinal.acceleration = -10.0;
```

Comment:
I think `0.0` is fine
```suggestion
    cmd.longitudinal.acceleration = 0.0;
```